### PR TITLE
25 24 fix audio

### PR DESCRIPTION
**Pull request by Skytrias**
Over the Typescript transition some of the issues i had fixed were not fully integrated. The changes pass the game over to the audio state which can then use methods to access game.sounds methods like changing all volume parameters etc.

I have tested this and it works fine.

### CODE REVIEW

File: src/renderer/ui/audio.js [hunk 1]
```diff
@@ -5,15 +5,16 @@ let audio_sfx_volume = 100
 let audio_msx_volume = 100
 let muted            = false
 
-function render(){
+function render(game){
```
Comment (omenking) [inline src/renderer/ui/audio.js]: game can be imported so its not necessary to pass along

**Comment by omenking** [comment]
For UI Audio you can import game at the top, and then remove passing game along.

File: src/renderer/ui/audio.js (hunk 1 above)
Comment (Skytrias) [reply src/renderer/ui/audio.js]: ill try that, should i change audio to typescript either?

Comment (omenking) [reply src/renderer/ui/audio.js]: Convert it all over to Typescript

**Comment by Skytrias** [comment]
the newest commit fixes everything youve requested along with other files switching to ts. mode.js is still a problem

**Pull request merged**